packages/locales/src/nl.ts

```diff
@@ -37,8 +35,8 @@ export const templates = {
   sd244b51f6d7de8e2: 'Voer een waarde in volgens het gevraagde patroon.',
   sd711c88b851db6c9: 'De volgende velden zijn incorrect:',
   sde6884478b7cc818: str`Voer tenminste ${0} karakters in (je hebt op dit momoment ${1} karakter${2}).`,
-  sea4f08110bb8f15d: 'verwijderen',
   see63aaad45b1b116: 'status',
   sf1ec4acb8d744ed9: 'Mededeling',
-  sf677da98fa27f9b6: 'Meer links'
+  sf677da98fa27f9b6: 'Meer links',
+  sf75c5ddbac2b60f9: str`Showing ${0} out of ${1} items`
```

Comment:
I don't think these changes are supposed to be there. I got them as well...
has there been some merging in the localisation file that generates this file?

Reply:
There were sth wrong with translations, I've just fixed/changed them.